.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "Tracking Task"
```

Comment:
Is "task" a catch-all for everything non-bug related? Seems like it, which makes sense, but just want to confirm

Reply:
Yup that's right ! Having a to create one for feature / task / etc seemed overkill at this point in time